docs/testcoverage.rst

```diff
@@ -0,0 +1,86 @@
+Test Coverage
+=============
+
+You can integrate Coverage.py with behave-django to find out the test coverage of your code.
+
+Dependencies
+------------
+
+At First, you should install Coverage.py dependency
+
+.. code-block:: bash
+
+    $ pip install coverage
+
+Environment.py
+--------------
+
+In ``environment.py``, add the code snippet below in the ``before_all`` function to start measuring test coverage:
+
+.. code-block:: python
+
+    import coverage
+
+    def before_all(context):
+        cov = coverage.Coverage()
+        cov.start()
+        context.cov = cov
+
+Then write below code to end up measuring test coverage.
+You can save the coverage result on html format.
+
+.. code-block:: python
+
+    def after_all(context):
+        cov = context.cov
+        cov.stop()
+        cov.save()
+        cov.html_report(directory="./cov")
```

Comment:
Note to self: This code could be placed in a [coverage module](https://github.com/behave/behave-django/blob/main/behave_django/coverage.py) (e.g. behave_django/coverage.py) and triggered by behave-django's management command, when needed, e.g. with a `--coverage` command line option.

Then, there would be no need to do copy+paste from the documentation to get coverage analysis running with your BDD tests.